torch/csrc/jit/passes/guard_elimination.cpp

```diff
@@ -1,3 +1,4 @@
+#include <caffe2/core/common.h>
```

Comment:
@Krovatkin any reason we need use "caffe2::make_unique" below?
Seems some places use "torch::make_unique", some use "c10::guts::make_unique" and some use "std::make_unique" directly. Is there a guideline to follow?

Reply:
std::make_unique is c++14 only. Just change it to c10::guts::make_unique or torch::make_unique and call it a day

Reply:
what Dima said. I needed to use unique ownership here since profiling embeds raw pointers into a graph and I just wanted to make sure we always know that they won't be accessed in some arbitrary way.